.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Related to #<ID>
+
+Acceptance criteria 
+=================== 
+- [ ] Functionality implemented in agreement with description of the ticket
+- [ ] Tests are updated and pass 
+- [ ] Code quality metrics ok 
```

Comment:
don't think we need this because if the pipeline fails then the request cannot be merged

Reply:
ok. removing.